docs/workflows/agile-development-process.md

```diff
@@ -0,0 +1,93 @@
+# Using Sigrid as part of an agile development process
+
+Sigrid supports all development process methodologies. That said, most organizations are either already using an [agile development process](https://en.wikipedia.org/wiki/Agile_software_development), or are currently in the process of migrating towards such a way of working. Working agile is therefore both the most mainstream and most desirable way of doing software development. 
+
+This page explains how Sigrid fits into a [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development) development process, which is the most popular and widely used version flavor of agile. Many of thes best practices outlined in this article can also be used when using other agile project management methodologies, such as [Kanban](https://en.wikipedia.org/wiki/Kanban).
```

Comment:
Instead of "widely used version flavor of agile" we can simplify to "widely used flavor of agile"